wor-paginate.gemspec

```diff
@@ -1,26 +1,29 @@
-$:.push File.expand_path("../lib", __FILE__)
-
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require "wor/paginate/version"
 require 'date'
 
 Gem::Specification.new do |s|
   s.name        = "wor-paginate"
   s.version     = Wor::Paginate::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.date        = Date.today
+  s.date        = '2017-06-23'
```

Comment:
Hmm Date.today sounds better, this way when we build the gem the date updates by itself